yarn.lock

```diff
@@ -1741,10 +1775,10 @@ istanbul-reports@^2.1.1:
   dependencies:
     handlebars "^4.1.2"
 
-italia-ts-commons@^5.1.3:
-  version "5.1.3"
-  resolved "https://registry.yarnpkg.com/italia-ts-commons/-/italia-ts-commons-5.1.3.tgz#1808aa61add9235c2b85db20297a7ec9e0e85d27"
-  integrity sha512-mo/PUxt2YqY4U+0XdZPcHloWIqk600CT/1V3Db3NXCCD5aC5uD/y4tfQdRqMZTwVLRxcwlPaOp7cZwHVaDtAjw==
+italia-ts-commons@5.1.1:
```

Comment:
why the downgrade?

Reply:
I had some troubles with version 5.1.3 and **io-ts** library 
Anyway I updated to the last version (5.1.9) and it seems to work properly
Done in https://github.com/teamdigitale/io-ops/pull/51/commits/9bef4300d28903ee84c8fbdeedc584a641d494f2